understanding/20/resize-text.html

```diff
@@ -15,31 +15,35 @@ <h2>Intent of Resize text</h2>
       
       <p>The intent of this Success Criterion is to ensure that visually rendered text, including
          text-based controls (text characters that have been displayed so that they can be
-         seen [vs. text characters that are still in data form such as ASCII]) can be scaled
+         seen, rather than text characters that are still in data form such as ASCII) can be scaled
```

Comment:
I fail to get "text characters that can be displayed so they can be seen, rather than text characters that are still in data form such as ASCII", and I am sure I won't be the only one.
Is the aim to exclude from scaling requirements things like an (i) icon which may be text or svg or rendered via font icon, since it would be considered a graphic? All of these can be "seen", i.e. this is about the visual display pf text anyway, and i a line be drawn it is probably one between text proper and text-based icons? Or am I missing the point entirely?

Reply:
admittedly, i had no clue what that bit is about, so left it untouched, apart from removing the horrible-looking `[vs. ...]` stuff

Reply:
i'd say this should also either be clarified or, if we can't work out anymore what it actually means, removed...

Reply:
removed the ASCII bit - see comment below https://github.com/w3c/wcag/pull/2270#issuecomment-1075082743